tendrl/monitoring_integration/__init__.py

```diff
@@ -0,0 +1,79 @@
+import __builtin__
+import traceback
+import sys
+import json
+
+
+import maps
+
+
+from tendrl.monitoring_integration.grafana import utils
+from tendrl.monitoring_integration.grafana import exceptions
+from tendrl.monitoring_integration.grafana import dashboard
+from tendrl.monitoring_integration.grafana import datasource
+
+
+def main():
+
+    # Creating Default Dashboards
+    dashboards = []
+    config = utils.get_conf("/etc/tendrl/monitoring-integration/" +
+                            "monitoring-integration.conf.yaml")
+    setattr(__builtin__, "NS", maps.NamedDict())
+    setattr(NS, "conf", config)
+
+    dashboards = dashboard.get_all_dashboards()
+
+    title = []
+
+    for dashboard_json in dashboards:
+        title.append(dashboard_json["uri"].split('/')[1])
+
+    for dashboard_json in config.dashboards:
+        if dashboard_json in title:
+            sys.stdout.write('\n' + "Dashboard " + str(dashboard_json) +
+                             " already exists" + '\n')
+            continue
+        response = dashboard.create_dashboard(dashboard_json)
```

Comment:
dashboards = dashboard.get_all_dashboards()
 
     title = []
 
     for dashboard_json in dashboards:
        title.append(dashboard_json["uri"].split('/')[1])
 
     for dashboard_json in config.dashboards:
         if dashboard_json in title:
             sys.stdout.write('\n' + "Dashboard " + str(dashboard_json) +
                              " already exists" + '\n')
             continue